### ISSUE TITLE
Feature request :  NGPE as Uno and a visual studio extension as a graphical viewer for nupkg file ?

### ISSUE THREAD

**Issue by fforjan** [NuGetPackageExplorer/NuGetPackageExplorer#1691]
At the moment, in visual studio code, it is painful to explorer a LOCALLY generated nuget package without publishing it.

But it is available as a web application. 

Could we simply package it as a web view control for a visual studio extension so that we can explorer locally generated (or download or...) from our project ?

**Comment by jeromelaban** [comment]
https://nuget.info allows for opening local files, is that what you're asking for?

**Comment by fforjan** [comment]
yes but directly integrated into visual studio code as an extension.
